doc/tutorials/setup_openstack_libvert.rst

```diff
@@ -0,0 +1,283 @@
+
+      ``lxc config edit openstack``
+
+      In the editor, add the following lines under **config**:
+
+      .. code-block:: none
+
+         linux.kernel_modules: iptable_nat, ip6table_nat, ebtables, openvswitch
+         raw.lxc: |-
+           lxc.cgroup.devices.allow = c 10:237 rwm
+           lxc.cgroup.devices.allow = b 7:* rwm
+           lxc.cgroup.devices.allow = c 243:0 rwm
+           lxc.mount.entry = /dev/net/tun dev/net/tun none bind,create=file 0 0
+           lxc.mount.auto=proc:rw sys:rw cgroup:rw
+         security.nesting: "true"
+         security.privileged: "true"
+
+      Save and exit the editor.
+
+   b. lxc config device add openstack eth1 nic name=eth1 nictype=bridged parent=acrn-br0
+   c. lxc config device add openstack acrn_vhm unix-char path=/dev/acrn_vhm
+   d. lxc config device add openstack loop-control unix-char path=/dev/loop-control
+   e. for n in {0..15}; do lxc config device add openstack loop$n unix-block path=/dev/loop$n; done;
+
+4. Launch the **openstack** container:
+
+   ``lxc start openstack``
+
+5. Log in to the **openstack** container:
+
+   ``lxc exec openstack -- su -l``
+
+6. Let ``systemd`` manage **eth1** in the container, with **eth0** as the
+   default route:
+
+   Edit ``/etc/netplan/50-cloud-init.yaml``
+
+   .. code-block:: none
+
+      network:
+
+          version: 2
+
+          ethernets:
+
+              eth0:
+
+                  dhcp4: true
+
+              eth1:
+
+                  dhcp4: true
+
+                  dhcp4-overrides:
+
+                      route-metric: 200
+
+
+7. Log out and restart the **openstack** container:
+
+   ``lxc restart openstack``
+
+8. Log in to the **openstack** container:
+
+   ``lxc exec openstack -- su -l``
+
+9. Set up the Intel proxy inside the **openstack** container via ``/etc/environment``
+   Make sure no_proxy is properly set up in ``/etc/environment`` inside the container. Both IP addresses assigned to **eth0** and **eth1** and their subnets must be included. For example:
+
+   ``no_proxy=intel.com,.intel.com,10.0.0.0/8,192.168.0.0/16,localhost,.local,127.0.0.0/8,134.134.0.0/16``
+
+10. Add a new user named **stack** and set permissions:
+
+    ``sudo useradd -s /bin/bash -d /opt/stack -m stack``
+
+    ``echo "stack ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers``
+
+11. Log out and restart the **openstack** container:
+
+    ``lxc restart openstack``
+
+The **openstack** container is now properly configured for OpenStack. Use ``lxc list`` to verify that both **eth0** and **eth1** appear in the container.
+
+Set up ACRN prerequisites inside the container
+**********************************************
+
+1. Log in to the **openstack** container as **stack** user:
+
+   ``lxc exec openstack -- su -l stack``
+
+2. Download and compile ACRN’s source code. Refer to :ref:`getting-started-building`.
+
+   - make
+   - cd misc/acrn-manager/; make
+   - Install only the user-space components (acrn-dm/acrnctl/acrnd)
```

Comment:
@peterfang Is this the best way to present lines 141-143? I do not understand what make refers to in line 141. Can we make this easier for the user?